config/settings.yml

```diff
@@ -12,6 +12,7 @@
           :critical:
 :ems_refresh:
   :vmware_cloud:
+    :inventory_object_refresh: true
```

Comment:
if old refresh is still support it, we should mock these values in specs

https://github.com/Ladas/manageiq-providers-azure/blob/812adcc147987999bc82b5c14aeec136dfb1e9f3/spec/models/manageiq/providers/azure/cloud_manager/refresher_spec.rb#L12

https://github.com/Ladas/manageiq-providers-azure/blob/949197815654ce8c926c8476511cb69ee3fc645d/spec/models/manageiq/providers/azure/cloud_manager/azure_refresher_spec_common.rb#L35

So usually we are testing old refresh and new refresh, then we can also test batch saving (using batch SQL queries to save inventory, it's good for larger inventories)